multisite-enhancements.php

```diff
@@ -134,6 +134,9 @@ public static function load() {
 		$file_base = self::$file_base;
 		define( 'MULTISITE_ENHANCEMENT_BASE', $file_base );
 
+		// Load configuration settings
+		require_once __DIR__ . '/settings.php';
+
 		$autoload_paths = glob( "$file_base/autoload/*.php" );
 
 		foreach ( $autoload_paths as $classnames => $path ) {
```

Comment:
We should also only load the classes there are active inside the option. So we need here again the method that get is a option active and his class necessary.

Reply:
👍 Yes, for most features we can simply skip loading the entire class. Implemented in https://github.com/bueltge/wordpress-multisite-enhancements/pull/63/commits/9640957e48534876aa2d53011dd6bcbce9a3d8b7